apps/availability-monitor/src/review/check.ts

```diff
@@ -42,12 +44,15 @@ export function createCheck<T, A extends unknown[]>(
       };
 
       try {
-        const [passed, value] = await retry(() => run(...args), {
-          retries: 2,
-          minTimeout: 500,
-          maxTimeout: 500,
-          maxRetryTime: 2e3,
-        });
+        log.verbose("running check %s for %s", check.id);
+        const [passed, value] = await retry(
+          () => withTimeout(() => run(...args), timeout),
+          {
+            retries: 2,
+            minTimeout: 500,
+            maxTimeout: 500,
```

Comment:
So does this essentially mean, that the timeout `of retry` does not work correctly??
because you have to set an additional one for the actual "retried" call?

Reply:
no, the timeout here adjusts the time between the retries, https://github.com/vercel/async-retry#api

Reply:
oh. nice to keep that in mind.